example/src/pages/components/form.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Page } from 'catalog';
+import { FormikInput } from 'june-design-system';
+
+export default class ButtonPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      type: 'text',
+      disabled: false,
+      field: {
+        name: 'example-input',
+      },
+      form: {
+        errors: { 'example-input': null },
+        touched: { 'example-input': false },
+      }
+    }
+  }
+
+  toggleTouched() {
+    this.setState({
+      form: {
+        ...this.state.form,
+        touched: {
+          'example-input': !this.state.form.touched['example-input'],
+        }
+      }
+    });
+  }
+
+  toggleError() {
+    this.setState({
+      form: {
+        ...this.state.form,
+        errors: {
+          'example-input': this.state.form.errors['example-input'] ? null : 'this input is incorrect',
+        }
+      }
+    });
+  }
+
+  changeDisable() {
+    this.setState({
+      disabled: !this.state.disabled
+    });
+  }
+
+  render() {
+    return (
+      <Page>
+        <h3>Form section title</h3>
+          <FormikInput
+            label="Email*"
```

Comment:
Ipv. zelf een * te zetten in het label kan je wss beter een aparte prop "required" toevoegen en deze gebruiken. (voor alle inputs)